### PR TITLE
Removing all references of `openfl_legacy`.

### PR DESCRIPTION
**Pull request by MAJigsaw77**

### CODE REVIEW

**Comment by Geokureli** [comment]
I don't think I truly understand the consequences of this change, and I can't find any resources of what openfl_legacy is or whether we should kill support of it

**Comment by MAJigsaw77** [comment]
This define had been removed since 2016 from openfl itself https://github.com/openfl/openfl/commit/15be4f5bd5eecbd8f0074ddfe966564d99ee1dc0

File: flixel/text/FlxText.hx [hunk 1]
```diff
@@ -1249,4 +1249,4 @@ enum abstract FlxTextAlign(String) from String
 	}
 }
 
-private typedef AlignType = #if openfl_legacy String #else TextFormatAlign #end;
+private typedef AlignType = TextFormatAlign;
```
Comment (Geokureli) [inline flixel/text/FlxText.hx:1252]: We should remove this type and use TextFormatAlign, since most functions use TextFormatAlign, already

**Review by Geokureli — CHANGES_REQUESTED** [review]
thanks for finding that removal, I asked in the openfl forum and no one replied

I think having AlignType adds confusion to FlxText, let's take it out. Rest looks good

File: flixel/text/FlxText.hx (hunk 1 above)
Comment (Geokureli) [reply flixel/text/FlxText.hx:1252]: ...unless you see any reason for it to exist

Comment (MAJigsaw77) [reply flixel/text/FlxText.hx:1252]: > We should remove this type and use TextFormatAlign, since most functions use TextFormatAlign, already

Ok

**Review by Geokureli — APPROVED** [review]

**Comment by Geokureli** [comment]
Thanks bro

**Pull request merged**